lib/database.js

```diff
@@ -28,14 +27,14 @@ function getAnswersById (id) {
     .catch((error) => console.log(error))
 }
 
-function addQuestion (content, title, created) {
+function addQuestion (content, title) {
   return knex('questions')
-    .insert({content, title, created})
+    .insert({content, title})
     .catch((error) => console.log(error))
```

Comment:
I mentioned this to Erwin on Friday, but I think we need to be a bit cautious that in case of a database error this catch block doesn't swallow the error, causing the request to hang (no response sent). It might be a case of returning a rejected promise with the error in it. Shouldn't stop this PR, but something to keep in mind...